packages/manager/core/application/src/ovh-routing.tsx

```diff
@@ -107,16 +115,29 @@ export function createAppRouter() {
   const appBlobKey = '/pages/_app.tsx';
   const appBlob = import.meta.glob<Module>('/pages/_app.tsx', { eager: true });
 
-  const routes = [...regularRoutes, { path: '*', element: <NotFound /> }];
+  const newRoutes = regularRoutes.map((route, index) => {
+    if (route.path === 'dashboard') {
+      return {
+        ...route,
+        path: '*',
+        errorElement: <NotFound />,
+        error: <NotFound />,
+      };
+    }
```

Comment:
**question:** Why is it different for `dashboard`?

Reply:
Its different for dashboard to stay iso as angular, to have the services name in first path 

https://www.ovh.com/manager/#/dedicated/nasha/{service}

and not  :

https://www.ovh.com/manager/#/dedicated/nasha/dashboard/{service}

the package generouted didn't allow us to us parameter in first path so we need to made a fix